### PR TITLE
FIX logloss name in sklearn

### PR DESCRIPTION
**Pull request by ceelestin**
Fixed the name of the loss of the SGDClassifier in solvers/sklearn.py due to the following error:

The 'loss' parameter of SGDClassifier must be a str among {'hinge', 'perceptron', 'log_loss', 'squared_hinge', 'huber', 'modified_huber', 'squared_error', 'epsilon_insensitive', 'squared_epsilon_insensitive'}. Got 'log' instead.

### CODE REVIEW

**Review by tomMoral — APPROVED** [review]
LGTM! Thx
